pulpcore.fc

```diff
@@ -20,6 +20,7 @@
 /var/lib/pulp/artifact(/.*)? 	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
 /var/lib/pulp/assets(/.*)? 	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
 /var/lib/pulp/devel(/.*)? 	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
+/var/lib/pulp/pulpcore_static(/.*)? 	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
```

Comment:
This should have the webserver label I believe:
`pulp_webserver_static_dir`.
https://github.com/pulp/pulp_installer/blob/master/roles/pulp_webserver/README.md#role-variables

The .well-known directory for letsencrypt is 1 usage, but I think @dkliban planned to use it for other stuff as well.

Reply:
Aha. Isn't it really a docroot to be served by `$webserver` then? That would imply the context should be `httpd_sys_rw_content_t` (or ro?).

I also think the name is rather ugly because it says nothing about the intent. In https://github.com/theforeman/puppet-pulpcore/pull/105 I went for docroot, after Apache's `DocumentRoot` parameter.

It would make my life a lot easier if these things were discussed with the Satellite Platform team as well. Our needs are very similar, but it feels like I'm always catching up by reading the code, when issues are raised or when I raise an issue myself.

Reply:
@ekohl I totally agree, next time we will discuss these folderpaths with you. I'm adding the need to do so in the future to the subteam agenda now.

Do you want to open a separate discussion on the folderpath that pulp_installer should use?

As for the context, httpd_sys_rw_content_t sounds right, it should work for nginx also. There does not seem to be an ro context. And the contents of the well-known directory get written by an ansible role, not by apache/nginx. apache/nginx just hosts it.

Reply:
@Spredzy ping, can you resume work on this?

Reply:
@mikedep333 before we merge this, do we want to define the definitive directory and then align the installers on it? I'd prefer to keep the policy minimal.

Reply:
@ekohl Yeah, the Pulp team method would be to [write an issue](https://pulp.plan.io), and we discuss it at our Pulp Installer subteam meeting, or a separate meeting. Because which dir to use is design work.

Reply:
https://pulp.plan.io/issues/7750

Reply:
Updated for suggested label, until ^ is talked about and decided